### DISCUSSION TITLE
Set enforce-secure-profile to false by default

### DISCUSSION THREAD

**Discussion by Zipdox2** [PaperMC/Paper#10864] [Ideas]
### Is your feature request related to a problem?

Everyone hates Minecraft chat reports, and many players have mods that disable them. Unfortunately this won't let them talk on servers with the default enforce-secure-profile setting.

### Describe the solution you'd like.

Set enforce-secure-profile to false by default.

### Describe alternatives you've considered.

No alternative possible.

**Comment by Owen1212055** [comment]
It's not in our best interest to deviate from vanilla defaults. Server admins can configure if they so please, but this is not something we are interested in.
